content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Setting up Oracle
+kind: documentation
+description: Setting up Database Monitoring on an Oracle database
+disable_sidebar: true
```

Comment:
```suggestion
disable_sidebar: true
private: true
is_beta: true
```

Reply:
Adding the above front matter will hid the article from appearing in search results. This is standard for private betas, although let me know if you'd prefer that it be discoverable via search.